lib/pages/home_page.dart

```diff
@@ -231,7 +244,9 @@ class _HomePageState extends State<HomePage> {
                   shape: const CircleBorder(),
                   padding: const EdgeInsets.all(12.0),
                 ),
-                child: const Text('start'),
+                child: isAlreadyStarted
+                    ? Text(displayDistance)
```

Comment:
Nějak vizuálně vylepšit, přeci jen se dostáváme do nějakého toho finiše tak ať to i nějak vypadá 😉